components/server/src/ome/security/CompositeACLVoter.java

```diff
@@ -100,6 +101,21 @@ public void throwUpdateViolation(IObject object) throws SecurityViolation {
         return choose().restrictions(object);
     }
 
+    @Override
+    public void setPermittedClasses(Map<Integer, Set<Class<? extends IObject>>> objectClassesPermitted) {
+        basic.setPermittedClasses(objectClassesPermitted);
```

Comment:
(In this case we could eventually break out a new permitted-classes configuration parsing bean that is a common dependency of the basic ACL voter and the graph request factory.)